realm-jni/build.gradle

```diff
@@ -87,15 +87,34 @@ if (project.forceDownloadCore instanceof String) {
     ext.forceDownloadCore = Boolean.parseBoolean(project.forceDownloadCore)
 }
 
+def getNdkDir() {
+    def ndkDir = null
+    if (project.ext.properties.containsKey('ndk.dir')) {
+        ndkDir = project.ext['ndk.dir']
+    } else if (System.getenv('ANDROID_NDK')) {
```

Comment:
I don't have a strong preference too. Although other app I installed needs `ANDROID_NDK`, I can set both `ANDROID_NDK` and `NDK_HOME` in my laptop.